### PR TITLE
testaddons fix

### PR DESCRIPTION
**Pull request by mozrokafor**
## Description

    Describe your changes

## Reference

    i.e Jira or Github issue URL

## Checklist
    
- [ ] My code follows the style guidelines for this project
- [ ] I have not added any packages that contain high risk or unknown licenses (GPL,  LGPL, MPL, etc. consult with DevOps if in question)
- [ ] I have performed a self review of my own code
- [ ] I have commented my code PARTICULARLY in hard to understand areas
- [ ] I have added thorough tests where needed

### CODE REVIEW

File: src/addons/manager/addonmanager.cpp [hunk 1]
```diff
@@ -15,6 +15,7 @@
 #include "tasks/addon/taskaddon.h"
 #include "tasks/function/taskfunction.h"
 #include "taskscheduler.h"
+#include "tasks/addonindex/taskaddonindex.h"
```
Comment (bakulf) [inline src/addons/manager/addonmanager.cpp:18]: alphabetical order

File: src/addons/manager/addonmanager.h [hunk 2]
```diff
@@ -62,6 +62,7 @@ class AddonManager final : public QAbstractListModel {
   void initialize();
 
   void updateAddonsList(QList<AddonData> addons);
+  void refreshAddons();
```
Comment (bakulf) [inline src/addons/manager/addonmanager.h:65]: Add an empty line after this method.

File: tests/functional/testAddons.js [hunk 3]
```diff
@@ -2,55 +2,67 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-const assert = require('assert');
-const vpn = require('./helper.js');
-
-describe('Addons', function() {
-  this.ctx.authenticationNeeded = true;
+ const assert = require('assert');
```
Comment (bakulf) [inline tests/functional/testAddons.js:5]: revert all of these changes.

**Review by bakulf — CHANGES_REQUESTED** [review]

**Comment by mozrokafor** [comment]
being addressed on https://github.com/mozilla-mobile/mozilla-vpn-client/pull/4922

**Pull request closed without merging**